docs/docs/legal/policies/rate-limit-policy.md

```diff
@@ -3,7 +3,7 @@ title: Rate Limit Policy
 custom_edit_url: null
```

Comment:
@stebenz do you know what this field is about?
Do we have to change something elsewere?